content/source/docs/cloud/api/workspace-resources.html.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: "cloud"
+page_title: "Workspaces - API Docs - Terraform Cloud and Terraform Enterprise"
+---
+
+[200]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200
+[201]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201
+[202]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/202
+[204]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/204
+[400]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400
+[401]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401
+[403]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/403
+[404]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404
+[409]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/409
+[412]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/412
+[422]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422
+[429]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/429
+[500]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/500
+[504]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/504
+[JSON API document]: /docs/cloud/api/index.html#json-api-documents
+[JSON API error object]: https://jsonapi.org/format/#error-objects
+
+# Workspace Resources API
+
+## List Workspace Resources
+
+`GET /workspaces/:workspace_id/resources`
+
+Parameter            | Description
+---------------------|------------
+`:workspace_id`      | The ID of the workspace to retrieve resources from. Obtain this from the [workspace settings](../workspaces/settings.html) or the [Show Workspace](./workspaces.html#show-workspace) endpoint.
+
+Status  | Response                                       | Reason
+--------|------------------------------------------------|----------
+[200][] | [JSON API document][] (`type: "resources"`)    | Request was successful
+[404][] | [JSON API error object][]                      | Workspace not found or user unauthorized to perform action
+
+### Query Parameters
+
+This endpoint supports pagination [with standard URL query parameters](./index.html#query-parameters). Remember to percent-encode `[` as `%5B` and `]` as `%5D` if your tooling doesn't automatically encode URLs.
+
+Parameter                   | Description
+----------------------------|------------
+`page[number]`              | **Optional.** If omitted, the endpoint will return the first page.
+`page[size]`                | **Optional.** If omitted, the endpoint will return 20 workspace resources per page.
+
+### Permissions
+
+In order to list resources, the user must have permission to read resources for the specified workspace. ([More about permissions.](/docs/cloud/users-teams-organizations/permissions.html))
```

Comment:
```suggestion
To list resources, the user must have permission to read resources for the specified workspace. ([More about permissions.](/docs/cloud/users-teams-organizations/permissions.html))
```

Reply:
Going to take this suggestion and then get rid of the comma splice 😄